### DISCUSSION TITLE
Engraving ends successfully but the job is not engraved completely up to the end

### DISCUSSION THREAD

**Discussion by guerravinagre** [arkypita/LaserGRBL#2444] [Q&A]
I have an Ortur LM2 20W and from the last 3 engraving jobs i keep having the same error.
The laser grbl succesfully completes the job and does not present any error, but the image is not completely engraved up to the end.
I have tried resetting the laser machine, restarting the PC, updating LAser grbl, but the problem still occurs...

![lsrgrbl finish](https://github.com/user-attachments/assets/0218253c-f7a3-4640-96e5-590ff31e2883)
![image comparison job](https://github.com/user-attachments/assets/127049a2-fd5d-425c-9c5e-092a06e0af28)

any help would trully be lifesaving by this time of year!!!

**Comment by Tribo-2dot0** [comment]
Está gravando a imagem completa mas de forma comprimida no sentido do eixo Y, parece que seus passos/mm nesse eixo estão configurados incorretamente. Quais valores você tem em $100 e $101?

**Reply by guerravinagre** [reply]
Update: following day after a good night rest of the laser and PC I did a burn test and everything was ok ! Then I perform the final burns or a real live wood I wanted and it was perfect !!! 
many thanks nevertheless. If it happens again I will look in your advice. Thanks
